tests/unit/s2n_self_talk_session_resumption_test.c

```diff
@@ -118,43 +282,118 @@ int main(int argc, char **argv)
     EXPECT_SUCCESS(close(client_to_server[1]));
     EXPECT_SUCCESS(close(server_to_client[0]));
 
-    EXPECT_NOT_NULL(conn = s2n_connection_new(S2N_SERVER));
-    conn->server_protocol_version = S2N_TLS12;
-    conn->client_protocol_version = S2N_TLS12;
-    conn->actual_protocol_version = S2N_TLS12;
+    /* Initial handshake */
+    {
+        EXPECT_NOT_NULL(conn = s2n_connection_new(S2N_SERVER));
+        conn->server_protocol_version = S2N_TLS12;
```

Comment:
same here, do we need to set S2N_TLS12? I thought it's default anyway.

Reply:
removed.

Reply:
Still there (:

Reply:
Will get rid of it.